### ISSUE TITLE
Expand and Update Docs for 0.2

### ISSUE THREAD

**Issue by jbouffard** [locationtech-labs/geopyspark#267]
Once `milestone/mvp` is merged into `master`, much of the docs for GeoPySpark will either outdated, wrong, or incomplete. Therefore, time needs to be taken to make sure everything is up-to-date and expanded as needed.

A few key things in particular that need addressing:

- Update demos to use the Scala side tile server.
- Explain the difference between the the Scala side server and the `read`/`lookup` methods
- Introduce the `ColorMap` and its uses
- Look into embedding jupyter notebooks as examples

Ideally, these docs should I either be done/near complete before we merge into master.

**Comment by jbouffard** [comment]
Resoled by #483 #490 #467 #462 #460 #459 #457 #452
